docs/operate/get-started/other-hardware/dependencies.md

```diff
@@ -172,46 +179,52 @@ If you need to maintain the state of your resource, see [(Optional) Create and e
 
 ### Optional dependencies
 
+If an optional dependency does not start, the module will continue to build and reconfigure without it.
+`viam-server` reattempts to construct the optional dependency every 5 seconds.
+When an optional dependency constructs successfully, your modular resource reconfigures so it can access the optional dependency.
```

Comment:
@benjirewis is this good/accurate?

Reply:
Looks pretty good to me! Left one more suggestion.